helm-charts/fluentd-es/templates/fluentd-es.yaml

```diff
@@ -56,6 +56,31 @@ roleRef:
   name: {{  template "fluentd-es.fullname" . }}
   apiGroup: ""
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fluentd-es-metrics
+  namespace: logging
+  labels:
+    k8s-app: fluentd-es
+    version: {{ .Values.image.tag }}
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
I believe `kubernetes.io/cluster-service` and `addonmanager.kubernetes.io/mode` won't do anything (the latter is probably a remnant of the past, the former might be significant for the actual `DaemonSet`). However, I suppose it's fine to leave them in for now - at least it's uniform across all resources.